libs/runtime/ebpf_hash_table.c

```diff
@@ -155,41 +155,76 @@ _ebpf_murmur3_32(_In_reads_((length_in_bits + 7) / 8) const uint8_t* key, size_t
 }
 
 /**
- * @brief Given two potentially non-comparable key values, extract the key and
- * compare them.
+ * @brief Compare keys assuming they are integers.
  *
  * @param[in] hash_table Hash table the keys belong to.
  * @param[in] key_a First key.
  * @param[in] key_b Second key.
- * @retval -1 if key_a < key_b
- * @retval 0 if key_a == key_b
- * @retval 1 if key_a > key_b
+ *
+ * @return -1 if key_a < key_b, 0 if key_a == key_b, 1 if key_a > key_b.
+ */
+static __forceinline int
```

Comment:
```suggestion
static __forceinline int32_t
```

Reply:
Given that memcmp is defined as returning int, so I want to keep it the same so the compiler can tail call to memcmp.